python/eos/data/pypmc.py

```diff
@@ -155,6 +156,49 @@ def create(path, density):
         with open(os.path.join(path, 'description.yaml'), 'w') as description_file:
             yaml.dump(description, description_file, default_flow_style=False)
 
+    @staticmethod
+    def _cartesian_product(densityA, densityB):
+        """ Compute the cartesian product of two mixture densities. The means and covariances
+        of the components are copied and concatenated and the weights are multiplied. Note that
+        this product is not commutative. Only Gaussian mixtures are supported.
```

Comment:
``... not commutative in the sense that the order of densities affects the order of parameters in the output density.``